nuxeo-core/nuxeo-core-event/src/test/java/org/nuxeo/ecm/core/work/TestWorkManagerImpl.java

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
What is the reason of this new file ?
what are you testing except that a sleep is working ?

Reply:
I test the added behavior, and created this new file because StreamWorkManagerTest extends WorkManagerTest and is not concerned by this test.

Reply:
Merged with the existing test and ignored in StreamWorkManagerTest